api/Location.json

```diff
@@ -406,7 +406,8 @@
               "notes": "Before Firefox 53, the <code>pathname</code> property returned wrong parts of the URL. For example, for a URL of http://z.com/x?a=true&b=false, <code>pathname</code> would return \"/x?a=true&b=false\" rather than \"/x\"."
             },
             "ie": {
-              "version_added": true
+              "version_added": true,
+              "notes": "IE11 does not provide the leading slash char."
```

Comment:
```suggestion
              "notes": "Internet Explorer does not provide the leading slash character in the <code>pathname</code> (<code>docs/Web/API/Location</code> instead of <code>/docs/Web/API/Location</code>)."
```

Reply:
Can confirm that this behavior also wasn't present in Edge 12, but I can also confirm this was present in IE 8 as well.  Let's update the wording of the note some to reflect all IE versions, and to spell out abbreviations such as "char"?

Reply:
I do not have the option to change the Edge version. :(

IE8
![ie8](https://user-images.githubusercontent.com/7441419/60782692-0380ca00-a148-11e9-95a1-6b78b26eab6a.png)

IE5
![ie5](https://user-images.githubusercontent.com/7441419/60782705-0aa7d800-a148-11e9-867f-bc25cac9a527.png)

Reply:
I use SauceLabs to do the testing within each browser at each version.  There's also BrowserStack, both of which come in handy for things like this!  (I also don't know how that compatibility dropdown works, but we've confirmed it already happens in older IE versions as well, so I'm not worried.  😛)